searcher.proto

```diff
@@ -58,7 +59,13 @@ message GetTipAccountsResponse {
   repeated string accounts = 1;
 }
 
+message SubscribeBundleResultsRequest {}
+
 service SearcherService {
+  // Searchers can invoke this endpoint to subscribe to their respective bundle results.
+  // A success result would indicate the bundle won its state auction and was submitted to the validator.
+  rpc SubscribeBundleResults (SubscribeBundleResultsRequest) returns (stream bundle.BundleResult) {}
```

Comment:
@buffalu perhaps make sense to have this be on a separate service type like `StatsService` or something so we're not married to it being implemented by the same service. In the short term could just have single service implement both interfaces. Thoughts?

Reply:
Something similar to this: https://github.com/jito-foundation/jito-relayer/blob/master/transaction-relayer/src/main.rs#L372

Reply:
i think this is fine. maybe add do v1 to method name?

Reply:
alternatively could do whatyou suggested and rename to mempool service or something else. however adds a little more complexity from clients